3.6/s2i/bin/assemble

```diff
@@ -28,7 +29,7 @@ shopt -s dotglob
 echo "---> Installing application source ..."
 mv /tmp/src/* ./
 
-if [[ ! -z "$UPGRADE_PIP_TO_LATEST" ]]; then
+if [[ ! -z "$UPGRADE_PIP_TO_LATEST" || ! -z "$ENABLE_PIPENV" ]]; then
```

Comment:
This is similar case to `pip install -U virtualenv` above. Update of pip from PyPI is not needed for 3.6.